example/books/README.md

```diff
@@ -0,0 +1,14 @@
+Change directory into the example directory, and compile the example
+capnproto schema:
+
+```
+cd $GOPATH/src/go-capnproto2/example/books
+capnp compile -I/home/johnk/go/src/capnproto.org/go/capnp/std/ -ogo books/books.capnp
+```
+
+Then build and run each example:
+```
+cd ex1 && go build . && ./bookstest1
+
+cd ex2 && go build . && capnp encode ../books/books.capnp Book < ./book.txt > ./book.capnpenc && ./bookstest2 < ./book.capnpenc && cd ..
```

Comment:
Minor thing, rather than creating a temporary file, you could just use a pipe:

```suggestion
cd ex2 && go build . && capnp encode ../books/books.capnp Book < ./book.txt | ./bookstest2 && cd ..
```